defaults/main.yml

```diff
@@ -27,3 +27,4 @@ packages:
   - google-fluentd
   - google-fluentd-catch-all-config
 stackdriver_collectd_service: stackdriver-agent
+stackdriver_openvas: False
```

Comment:
Please change False to false

Reply:
Done added information inside the readme. If you merge inside the dedicated branch v6 than we will have nothing to change inside the techops repo. Please merge the PR.